build.gradle

```diff
@@ -19,6 +19,7 @@ plugins {
     id 'org.springframework.boot' version '2.3.9.RELEASE'
     id "org.flywaydb.flyway" version "7.0.2"
     id 'au.com.dius.pact' version '4.1.7'
+    id 'io.freefair.lombok' version '5.3.0'
```

Comment:
How does this differ to the `org.projectlombok` dependency we are using?

Reply:
it is good to have this as we may use to check lombok generated object.